<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="M_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>字母A-Z或a-z</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="C_text">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>字母A-Z或a-z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Encrypt_Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decrypt_Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QLabel" name="M_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>明文M</string>
    </property>
   </widget>
   <widget class="QLabel" name="C_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>密文C</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="K_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>   数字</string>
    </property>
   </widget>
   <widget class="QLabel" name="K_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密钥K</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Encrypt_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>MtoC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Decrypt_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CtoM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MtoC()</slot>
  <slot>CtoM()</slot>
 </slots>
</ui>
